core/domain/topic_domain.py

```diff
@@ -1969,12 +2238,15 @@ def to_dict(self):
         """
         return {
             'topic_id': self.id,
-            'manager_names': user_services.get_human_readable_user_ids(
+            'manager_names': user_services.get_human_readable_user_ids( # type: ignore[no-untyped-call]
                 self.manager_ids),
             'topic_is_published': self.topic_is_published
         }
 
-    def is_manager(self, user_id):
+    def is_manager(
+        self,
+        user_id: Optional[str]
```

Comment:
Why optional?

Reply:
First I did it from docstring then I traced used_id in a few files and from [here](https://github.com/oppia/oppia/blob/86c025e2aecd3e3effb3bab558ed3952094560a2/core/controllers/base.py#L233)
probably, we can set `user_id` to `str`. Please correct me if I am wrong.

Reply:
Yes, let's force it to str here.

Reply:
hey, I tried to do it. Please take a look and let me know if you want me to change anything